lib/domain/repositories/firebase_auth_repository.dart

```diff
@@ -4,5 +4,6 @@ import 'package:firebase_auth/firebase_auth.dart';
 import '../../data/failure.dart';
 
 abstract class FirebaseAuthRepository {
+  bool get isLoggedIn;
```

Comment:
Logout also needs to be considered here

Reply:
Done. Done @Harryalways317 will implement it.